ecommerce/mail_api.py

```diff
@@ -67,3 +70,27 @@ def send_ecommerce_order_receipt(order_record):
 
     except:  # pylint: disable=bare-except
         log.exception("Error sending order receipt email.")
+
+
+def send_ecommerce_refund_message(order_record):
+    try:
```

Comment:
I think we should start this `try` block before the context manager. The following 3 lines won't seem to be troubling.